### PR TITLE
Adding Dyno Protocol

### PR DESCRIPTION
**Pull request by blockchainreg**

### CODE REVIEW

**Review by blockchainreg — COMMENTED** [review]
Updated

**Review by blockchainreg — COMMENTED** [review]
Updated

File: _data/chains/eip155-3966.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "DYNO Mainnet",
+  "chain": "DYNO",
+  "rpc": [
+    "https://api.dynoprotocol.com",
```
Comment (ligi) [inline _data/chains/eip155-3966.json:5]:
```suggestion
    "https://api.dynoprotocol.com"
```

File: _data/chains/eip155-3967.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "DYNO Testnet",
+  "chain": "DYNO",
+  "rpc": [
+    "https://tapi.dynoprotocol.com",
```
Comment (ligi) [inline _data/chains/eip155-3967.json:5]:
```suggestion
    "https://tapi.dynoprotocol.com"
```

**Review by ligi — CHANGES_REQUESTED** [review]
JSON is not valid

**Comment by blockchainreg** [comment]
> JSON is not valid

Updated suggested JSON

**Review by blockchainreg — COMMENTED** [review]
Updated suggested JSON

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `dyno` more than once

**Comment by blockchainreg** [comment]
> short nam

Thats weird i looked on chainlist before submitting i didnt see any dyno. ill think of something else real quick

**Comment by ligi** [comment]
you use it for 2 chains ^^

**Comment by blockchainreg** [comment]
> you use it for 2 chains ^^

Got it my apologies  when needing sleep goes wrong. lol

**Comment by ligi** [comment]
;-)

Please also consider supporting our grant: https://gitcoin.co/grants/2974/ethereum-lists

**Comment by blockchainreg** [comment]
> ;-)
> 
> Please also consider supporting our grant: https://gitcoin.co/grants/2974/ethereum-lists

I certainley will. you been managing this repo a long time. Thanks for your dedication!

**Comment by blockchainreg** [comment]
> ;-)
> 
> Please also consider supporting our grant: https://gitcoin.co/grants/2974/ethereum-lists

Quick question. On chainlist when I try to add the network it gives me this error  https://imgur.com/d6WO1fw   How do I fix It?

**Comment by ligi** [comment]
fix the chainID - either in the node or here - see: 

```
✓ ~ curl -X POST -H  "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_chainId","params":[],"id":1}' https://tapi.dynoprotocol.com
{
  "jsonrpc" : "2.0",
  "id" : 1,
  "result" : "0x2c4"
}⏎        
```

**Pull request merged**